Temporarily disable acrn-crashlog until we can integrate it properly.

Upstream-Status: Inappropriate
Signed-off-by: Ross Burton <ross.burton@intel.com>

From edea6fe6dee98f67fd4669eed2e1d730e61adb55 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@intel.com>
Date: Fri, 15 Mar 2019 12:20:16 +0000
Subject: [PATCH 1/2] disable crashlog

---
 tools/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/misc/Makefile b/misc/Makefile
index de7ecdae..56475368 100644
--- a/misc/Makefile
+++ b/misc/Makefile
@@ -4,7 +4,7 @@ RELEASE ?= 0

 .PHONY: all acrn-crashlog acrnlog acrn-manager acrntrace acrnbridge
 ifeq ($(RELEASE),0)
-all: acrn-crashlog acrnlog acrn-manager acrntrace acrnbridge
+all: acrnlog acrn-manager acrntrace acrnbridge
 else
 all: acrn-manager acrnbridge
 endif
@@ -34,7 +34,7 @@ clean:

 .PHONY: install
 ifeq ($(RELEASE),0)
-install: acrn-crashlog-install acrnlog-install acrn-manager-install acrntrace-install acrnbridge-install
+install: acrnlog-install acrn-manager-install acrntrace-install acrnbridge-install
 else
 install: acrn-manager-install acrnbridge-install
 endif
--
2.11.0
